Tests/FoundationTests/UserDefaultsExtensionsTests.swift

```diff
@@ -37,5 +41,13 @@ final class UserDefaultsExtensionsTests: XCTestCase {
 		XCTAssertNotNil(UserDefaults.standard.date(forKey: key))
 		XCTAssertEqual(UserDefaults.standard.date(forKey: key)!, date)
 	}
+    
+    func testCodable() {
```

Comment:
We have to split this into two tests: one for set, one for get

Reply:
Noted, will update

Reply:
Question, what would be my assertion for the set test since verifying if the object was stored correctly is thru the get test?

Reply:
@jason-ingenuity I really think the tests are the same. It probably just needs to be duplicated for the sake of consistency

Reply:
I see. Will do